### ISSUE TITLE
F52 indicates in the title it's a failure of SC 3.2.1 although the failure is no longer  mapped to SC  3.2.1

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1262]
The name of the technique should be changed if it is no longer relevant to 3.2.1.
https://www.w3.org/WAI/WCAG21/Techniques/failures/F52

Failure of Success Criterion 3.2.1 and 3.2.5 due to opening a new window as soon as a new page is loade
